src/main/java/games/strategy/triplea/ui/AbstractUIContext.java

```diff
@@ -34,43 +34,39 @@
   protected static final String UNIT_SCALE_PREF = "UnitScale";
   protected static final String MAP_SKIN_PREF = "MapSkin";
   protected static final String MAP_SCALE_PREF = "MapScale";
-  protected static final Logger s_logger = Logger.getLogger(AbstractUIContext.class.getName());
-  protected static String m_mapDir;
+  protected static final Logger logger = Logger.getLogger(AbstractUIContext.class.getName());
+  protected static String mapDir;
   protected static final String LOCK_MAP = "LockMap";
   protected static final String SHOW_END_OF_TURN_REPORT = "ShowEndOfTurnReport";
   protected static final String SHOW_TRIGGERED_NOTIFICATIONS = "ShowTriggeredNotifications";
   protected static final String SHOW_TRIGGERED_CHANCE_SUCCESSFUL = "ShowTriggeredChanceSuccessful";
   protected static final String SHOW_TRIGGERED_CHANCE_FAILURE = "ShowTriggeredChanceFailure";
   protected static final String SHOW_BATTLES_BETWEEN_AIS = "ShowBattlesBetweenAIs";
-  protected static ResourceLoader m_resourceLoader;
-  // instance
-  protected boolean m_isShutDown = false;
-  protected final List<Window> m_windowsToCloseOnShutdown = new ArrayList<>();
-  protected final List<Active> m_activeToDeactivate = new ArrayList<>();
-  protected final CountDownLatchHandler m_latchesToCloseOnShutdown = new CountDownLatchHandler(false);
-  protected LocalPlayers m_localPlayers;
-  protected double m_scale = 1;
+  protected static ResourceLoader resourceLoader;
+
+  protected boolean isShutDown = false;
+  protected final List<Window> windowsToCloseOnShutdown = new ArrayList<>();
+  protected final List<Active> activeToDeactivate = new ArrayList<>();
+  protected final CountDownLatchHandler latchesToCloseOnShutdown = new CountDownLatchHandler(false);
+  protected LocalPlayers localPlayers;
+  protected double scale = 1;
 
   public static ResourceLoader getResourceLoader() {
-    return m_resourceLoader;
+    return resourceLoader;
   }
 
-  public static int getAIPauseDuration() {
+  public static int getAiPauseDuration() {
     return ClientContext.aiSettings().getAiPauseDuration();
   }
 
-  public static void setAIPauseDuration(final int value) {
```

Comment:
This method had no callers, so I removed it rather than renaming it to fix the violation.